app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,278 @@
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.getSession('current');
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    An authenticated session in appwrite lasts for 1 year and is then automatically destroyed.
+</p>
+
+<h2><a href="/docs/authentication#persistence" id="persistence">Persistence</a></h2>
+
+<p>
+    Appwrite handles the persistence of the session in a consistent way across SDKs. After authenticating with an SDK, the SDK will persist the session so that the user will not need to log in again the next time they open the app. How the SDK persists in the session depends on the SDK.
+</p>
+
+<ul>
+    <li class="margin-bottom-tiny">Web - Uses a session secure cookie and falls back to local storage when a session cookie is not available.</li>
+    <li class="margin-bottom-tiny">Flutter - Uses a session cookie stored in Application Documents through the path_provider package.</li>
+    <li class="margin-bottom-tiny">Apple - Uses a session cookie.</li>
+    <li class="margin-bottom-tiny">Android - Uses a session cookie stored in SharedPreferences.</li>
+</ul>
+
+<h2><a href="/docs/authentication#security" id="security">Security</a></h2>
+
+<p>
+    Security is very important to protect users' data and privacy. Appwrite uses a permissions model coupled with user sessions to ensure users only have access to certain information based on the permissions. With Appwrite services, including databases and storage, access is granted to documents and files at the collection, bucket, document, or file level. This access is consistent across access to these items in relation to document access, file access, and real-time events.
+</p>
+
+<h2><a href="/docs/authentication#best-practices" id="best-practices">Best Practices</a></h2>
+
+<p>
+    Here are some common best practices when handling sessions and keys used for authentication purposes.
+</p>
+
+<ul>
+    <li class="margin-bottom-tiny">Only keep user sessions active as long as needed.</li>
+    <li class="margin-bottom-tiny">Access Appwrite via HTTPS to prevent any redirects from interfering with your requests.</li>
+    <li class="margin-bottom-tiny">Run Appwrite on standard ports to ensure everything works correctly. The complete endpoint is located in the Appwrite Console</li>
+    <li class="margin-bottom-tiny">Prefer using OAuth2 for authentication with first-party redirects</li>
+    <li class="margin-bottom-tiny">Generate an SSL certification for the Appwrite server</li>
```

Comment:
```suggestion
    <li class="margin-bottom-tiny">Generate an SSL certification for the Appwrite instance</li>
```

Reply:
We agreed during writing the terminology guidelines to say "Appwrite Instance" for self-deployed.